### PR TITLE
Add gnocchi

### PR DESCRIPTION
**Pull request by victor**
Purpose of this branch is lalalala

### CODE REVIEW

File: menu.txt [hunk 1]
```diff
@@ -3,9 +3,10 @@ STARTERS
 * Salad
 * Spaghetti
 * Pasta
-* Pizza
+* Pizza
```
Comment (juliagarrigos) [inline menu.txt]: Why did you add an space here?

Comment (victor) [inline menu.txt]: I don't know

Comment (juliagarrigos) [inline menu.txt]: 👍
